syntax = "proto3";
package protos.playbooks;

import "google/protobuf/wrappers.proto";
import "protos/base.proto";

message PagerdutyNotesWorkflowAction {
  google.protobuf.StringValue incident_id = 1;
}
